index.js

```diff
@@ -120,6 +120,8 @@ const updatePage = (result) => {
 }
 
 const buildQuery = () => {
+    // What are some resons for doing `querySelector` from the `form` object
```

Comment:
Totally an artifact of the direction I was going to go and not refactoring after I changed directions. I was thinking about having a toggle for extra form items like country, and some of the other query parameters available to the api, then I was going to do a "queryselectorAll('input')' and then map (well foreach, as map doesn't work on a nodeList) it out to make the query string. Then I thought, maybe just select each one because I know this app isn't going to be able to add form items. I started backtracking and didn't use the more efficient (getElementById).